src/azure-cli-testsdk/azure/cli/testsdk/patches.py

```diff
@@ -3,6 +3,8 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from azure_devtools.scenario_tests.patches import patch_time_sleep_api, _mock_in_unit_test
```

Comment:
If it is indeed needed and exposed from devtools, it should be renamed since it is not private anymore.